6_visualize/src/create_gif_makefile.R

```diff
@@ -7,8 +7,7 @@ create_intro_gif_makefile <- function(makefile, task_plan, remake_file) {
       '6_visualize/src/prep_spark_line_fun.R',
       '6_visualize/src/prep_gage2spark_fun.R',
       '6_visualize/src/create_animation_frame.R'),
-    file_extensions=c('feather','ind'),
-    ind_complete=TRUE)
```

Comment:
Well, I think the output of the indicator file having all the filenames for PNGs so that we can use those to build the GIF is also an outcome of the task_combiners branch.

Reply:
Right - but that's a new opportunity rather than a required change, right?

Reply:
Oh yes